crates/stash-api/src/sync/fetcher.rs

```diff
@@ -63,6 +63,7 @@ pub(crate) fn start_fetcher(
     scheduler_tx: Sender<SchedulerMessage>,
 ) -> std::thread::JoinHandle<()> {
     std::thread::spawn(move || {
+        // Should this be prefixed with POE_ or is unused currently?
         let client_id = std::env::var("CLIENT_ID").unwrap();
```

Comment:
Good question! `stash-api/src/sync` was the first, sync version I built based on threads as an exercise but I simply have not deleted it yet. This whole module should essentially just be removed.